src/Sulu/Bundle/AdminBundle/Resources/js/views/Form/toolbarActions/DeleteToolbarAction.js

```diff
@@ -176,7 +176,10 @@ export default class DeleteToolbarAction extends AbstractFormToolbarAction {
         const {id} = this.resourceFormStore;
 
         const visibleConditionFulfilled = !visibleCondition || jexl.evalSync(visibleCondition, this.conditionData);
-        const isDisabled = !id || (deleteLocale && jexl.evalSync('contentLocales.length == 1', this.conditionData));
+        const isDisabled = !id || (deleteLocale && jexl.evalSync(
+            'contentLocales && contentLocales|length == 1',
```

Comment:
@wachterjohannes `|` instead of `.`?

Reply:
@chirimoya yes - thats the syntax of jexl